### DISCUSSION TITLE
Tab autocomplete is not working in Markdown Files.

### DISCUSSION THREAD

**Discussion by JewinV** [community/community#11055] [Copilot Conversations] [answered]
I am using the Copilot extension in vs code. `Tab` autocomplete is working fine with python files but not with markdown files. Copilot is suggesting lines in *.md files but I can't autocomplete or `Ctrl + Enter`

Any clues on what could be wrong would be really helpful.

Thanks

**Comment by NickCrews** [comment]
I have the same issue with *.rst files. The faded suggestion appears, and it even says "Accept (Tab)" when I hover over the suggestion. But pressing tab actually just inserts a tab. Clicking "Accept (Tab)" does actually insert the suggest though.

![image](https://user-images.githubusercontent.com/10820686/153049862-9fe29d66-4746-4373-8b19-4a4cba91a12a.png)

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/7861#discussioncomment-1666029

**Reply by NickCrews** [reply]
Indeed. Thanks for the pointer. The solution for .md files is at https://github.com/yzhang-gh/vscode-markdown/issues/1011, and based on that solution, my adjustment for .rst files is this (added this to my `keybindings.json`):

```
// Place your key bindings in this file to override the defaults
[
    // Disable tab keybinding for restructured text, so that copilot can take over.
    // similar to https://github.com/yzhang-gh/vscode-markdown/issues/1011
    {
        "key": "tab",
        "command": "restructuredtext.editor.onTabKey",
        "when": "editorTextFocus && !inlineSuggestionVisible && !editorReadonly && !editorTabMovesFocus && !hasOtherSuggestions && !hasSnippetCompletions && !inSnippetMode && !suggestWidgetVisible && editorLangId == 'restructuredtext'"
      },
      {
        "key": "tab",
        "command": "-restructuredtext.editor.onTabKey",
        "when": "editorTextFocus && !editorReadonly && !editorTabMovesFocus && !hasOtherSuggestions && !hasSnippetCompletions && !inSnippetMode && !suggestWidgetVisible && editorLangId == 'restructuredtext'"
    },
    {
        "key": "tab",
        "command": "restructuredtext.editor.listEditing.onTabKey",
        "when": "editorTextFocus && !inlineSuggestionVisible && !editorReadonly && !editorTabMovesFocus && !hasOtherSuggestions && !hasSnippetCompletions && !inSnippetMode && !suggestWidgetVisible && editorLangId == 'restructuredtext'"
    },
    {
        "key": "tab",
        "command": "-restructuredtext.editor.listEditing.onTabKey",
        "when": "editorTextFocus && !editorReadonly && !editorTabMovesFocus && !hasOtherSuggestions && !hasSnippetCompletions && !inSnippetMode && !suggestWidgetVisible && editorLangId == 'restructuredtext'"
    },
]
```
Also made a PR to fix this with the restructuredtext extension at https://github.com/vscode-restructuredtext/vscode-restructuredtext/pull/345

**Reply by NickCrews** [reply]
@JewinV that linked issue has the solution you're looking for
